File: leetcode/Compute Alternating Sum/1.java
class Solution {
    public int alternatingSum(int[] nums) {
        int result = 0;
        boolean plus = true;
        for (int i = 0; i < nums.length; i++) {
            result += (plus ? nums[i] : -nums[i]);
            plus = !plus;
        }
        return result;
    }
}

File: leetcode/Rotate Array/1.kt
class Solution {
    fun rotate(nums: IntArray, k: Int): Unit {
        val result = IntArray(nums.size)
        for (i in nums.indices) {
            val idx = (i + k) % nums.size
            result[idx] = nums[i]
        }
        for (i in nums.indices) {
            nums[i] = result[i]
        }
    }
}